automation/src/test/scala/org/broadinstitute/dsde/test/api/SamApiSpec.scala

```diff
@@ -159,15 +159,15 @@ class SamApiSpec extends FreeSpec with BillingFixtures with Matchers with ScalaF
       val expectedProxyEmail1 = s"${username1}_$userId1@${Config.GCS.appsDomain}"
 */
       val expectedProxyEmail1 = s"$userId1@${Config.GCS.appsDomain}"
-/**/
+// /**/
```

Comment:
commented comment?

Reply:
`/**/` confuses github and makes everything below it appear commented out so i just commented out that comment to negate that effect. i'll change it to something that looks less stupid